administrator/language/de-DE/com_admin.ini

```
@@ -143,6 +143,8 @@ COM_ADMIN_POSTINSTALL_MSG_BEHIND_LOAD_BALANCER_DESCRIPTION="<p>Für Joomla-Seite
 COM_ADMIN_POSTINSTALL_MSG_BEHIND_LOAD_BALANCER_TITLE="Neue Konfigurationseinstellung „Load Balancer aktivieren“"
 COM_ADMIN_POSTINSTALL_MSG_HTACCESS_AUTOINDEX_DESCRIPTION="<p>Vor 3.9.22 enthielt die Standarddatei <strong>htaccess.txt</strong> fehlerhaften Code zum Deaktivieren von Verzeichnisauflistungen. Das Sicherheitsteam empfiehlt, die notwendigen Änderungen manuell auf jede bestehende .htaccess-Datei anzuwenden, da diese Datei nicht automatisch aktualisiert werden kann.</p><p>Alter Code:</p><pre>&lt;IfModule autoindex&gt;\n  IndexIgnore *\n&lt;/IfModule&gt;</pre><p>Neuer Code:</p><pre>&lt;IfModule mod_autoindex.c&gt;\n  IndexIgnore *\n&lt;/IfModule&gt;</pre>"
 COM_ADMIN_POSTINSTALL_MSG_HTACCESS_AUTOINDEX_TITLE=".htaccess Aktualisierung in Bezug auf Verzeichniseinträge"
+COM_ADMIN_POSTINSTALL_MSG_HTACCESS_SETCE_DESCRIPTION="<p>Vor 4.2.9 enthielt die Standarddatei <strong>htaccess.txt</strong> einen fehlerhaften Code für das Anhängen des HTTP-Headers "Content-Encoding". Dies konnte zu doppelten Kodierungsfehlern führen, wenn Joomla in einem Unterverzeichnis installiert ist und sowohl dieses Verzeichnis als auch das übergeordnete Verzeichnis eine .htaccess-Datei mit diesem Code enthalten. Die notwendigen Änderungen sollten manuell an einer bestehenden .htaccess-Datei vorgenommen werden, da diese Datei nicht automatisch aktualisiert werden kann.</p><p>Alter Code:</p><pre>Header append Content-Encoding gzip</pre><p>Neuer Code:</p><pre>Header set Content-Encoding gzip</pre>"
```

```suggestion
COM_ADMIN_POSTINSTALL_MSG_HTACCESS_SETCE_DESCRIPTION="<p>Vor 4.2.9 enthielt die Standarddatei <strong>htaccess.txt</strong> einen fehlerhaften Code für das Anhängen des HTTP-Headers &lt;Content-Encoding&gt;. Dies konnte zu doppelten Kodierungsfehlern führen, wenn Joomla in einem Unterverzeichnis installiert ist und sowohl dieses Verzeichnis als auch das übergeordnete Verzeichnis eine .htaccess-Datei mit diesem Code enthalten. Die notwendigen Änderungen sollten manuell an einer bestehenden .htaccess-Datei vorgenommen werden, da diese Datei nicht automatisch aktualisiert werden kann.</p><p>Alter Code:</p><pre>Header append Content-Encoding gzip</pre><p>Neuer Code:</p><pre>Header set Content-Encoding gzip</pre>"
```

Müssten die Anführungszeichn nicht als html code geschrieben werden?